src/api/system/system.ts

```diff
@@ -238,6 +238,15 @@ export default class System extends Base {
     }
 
     /**
+     * Retrieves system information.
+     * @tutorial system.getHostSpecs
+     * @return {Promise.<Object>}
```

Comment:
\<Object\> should be typed

Reply:
@rdepena What do you mean by object should be typed ?